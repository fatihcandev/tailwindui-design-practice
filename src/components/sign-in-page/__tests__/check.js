import React from 'react'
import renderer from 'react-test-renderer'

import Check from '../icons/check'

describe('Check', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Check />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
