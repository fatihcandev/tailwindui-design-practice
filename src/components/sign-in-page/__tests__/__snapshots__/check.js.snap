// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check renders correctly 1`] = `
<svg
  fill="none"
  stroke="currentColor"
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth="2"
  viewBox="0 0 24 24"
>
  <path
    d="M5 13l4 4L19 7"
  />
</svg>
`;
